github/src/main/resources/plugin-settings.template.html

```diff
@@ -29,6 +29,21 @@
     <span class="form_error" ng-show="GOINPUTNAME[oauth_token].$error.server">{{ GOINPUTNAME[oauth_token].$error.server }}</span>
 </div>
 <div class="form_item_block">
+    <label>GitHub Authorization Url:</label>
+    <input type="text" ng-model="authorize_url" ng-required="true" placeholder="https://github.com/login/oauth/authorize"/>
+    <span class="form_error" ng-show="GOINPUTNAME[authorize_url].$error.server">{{ GOINPUTNAME[authorize_url].$error.server }}</span>
+</div>
+<div class="form_item_block">
+    <label>GitHub Access Token Url:</label>
+    <input type="text" ng-model="access_token_url" ng-required="true" placeholder="https://github.com/login/oauth/access_token"/>
+    <span class="form_error" ng-show="GOINPUTNAME[access_token_url].$error.server">{{ GOINPUTNAME[access_token_url].$error.server }}</span>
+</div>
+<div class="form_item_block">
+    <label>GitHub API Url:</label>
+    <input type="text" ng-model="api_url" ng-required="true" placeholder="https://github.com/api/v3"/>
+    <span class="form_error" ng-show="GOINPUTNAME[api_url].$error.server">{{ GOINPUTNAME[api_url].$error.server }}</span>
+</div>
+<div class="form_item_block">
```

Comment:
Since these fields are optional, and are only used on GH enterprise, could you mark the label with `required for GH enterprise` and change the placeholder text to use `https://github.companyname.com/...` instead of `github.com` to avoid confusion?